### PR TITLE
Made Changes and created Seperate Contribute.md file

### PR DESCRIPTION
**Pull request by Yash1256**
Solving Issue #142

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096955&bid=16766381)

File: Contribution.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Contributing to whatsapp-play
+
+The first step is to create a fork and clone, then you can solve our [issues](https://github.com/rpotter12/whatsapp-play/issues) and help us find new ones. To debug with Visual Studio Code it is necessary to create a launcher with the arguments. <br />
+
+**Make sure you follow below guidelines before contributing.**
+
+1. Raise an issue before sending any PR.
+2. Make you changes to feature branch.
```
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096955)

File: Contribution.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+3. See if there is already an open PR for the same issue.
+
+**To create a launcher with arguments follow the steps bellow:** <br />
+1. Click in 'Debug' tab
+1. Click in 'Add Configuration'
+1. Select 'Module'
+1. Type 'wplay' and press Enter
+1. A json file will be opened. Inside configurations add the args, for example: "args":["-wb","name"]
```
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096955)

Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096955)

File: Contribution.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+## Contributing to whatsapp-play
+
+The first step is to create a fork and clone, then you can solve our [issues](https://github.com/rpotter12/whatsapp-play/issues) and help us find new ones. To debug with Visual Studio Code it is necessary to create a launcher with the arguments. <br />
+
+**Make sure you follow below guidelines before contributing.**
+
+1. Raise an issue before sending any PR.
+2. Make you changes to feature branch.
+3. See if there is already an open PR for the same issue.
```
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096955)

File: Contribution.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+## Contributing to whatsapp-play
+
+The first step is to create a fork and clone, then you can solve our [issues](https://github.com/rpotter12/whatsapp-play/issues) and help us find new ones. To debug with Visual Studio Code it is necessary to create a launcher with the arguments. <br />
+
+**Make sure you follow below guidelines before contributing.**
+
+1. Raise an issue before sending any PR.
```
Comment (rpotter12) [inline Contribution.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5096955)

**Comment by xandao6** [comment]
This issue was assigned to @Antaripa, I will see how to proceed.

**Pull request closed without merging**